### PR TITLE
Feature/sidesheet size

### PR DESCRIPTION
**Pull request by ingridklepsvik**

### CODE REVIEW

File: src/components/core/Header/styles.less [hunk 1]
```diff
@@ -15,6 +16,12 @@
     right: 0;
     z-index: 2;
 
+    @media (max-width: @mobile-max-width) {
+        grid-template-areas: "start logo aside" "content content content";
+        grid-template-columns: max-content max-content max-content;
```
Comment (eskil0312) [inline src/components/core/Header/styles.less]: Should this be added to this branch?

File: src/components/general/SideSheet/Modal/index.tsx [hunk 2]
```diff
@@ -10,21 +10,25 @@ import {
...
 
+export type SideSheetSize = 'xlarge' | 'large' | 'medium' | 'small';
+
 type ModalSideSheetProps = {
     children: ReactNode;
     header?: string;
     show?: boolean;
     onClose?: () => void;
     headerIcons?: ReactNode[];
+    size?: SideSheetSize;
```
Comment (eskil0312) [inline src/components/general/SideSheet/Modal/index.tsx]: Maybe we should default to xLarge so existing modal side sheets don't unexpectedly change size

File: src/components/core/Header/styles.less (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/core/Header/styles.less]: Removed it

File: src/components/general/SideSheet/Modal/index.tsx (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/general/SideSheet/Modal/index.tsx]: default to xLarge

File: src/components/core/Header/styles.less [hunk 3]
```diff
@@ -1,4 +1,5 @@
 @import "~styles/mixins.less";
+@import '~styles/variables.less';
```
[the remark this answers is not in the document]
Comment (ingridklepsvik) [reply src/components/core/Header/styles.less]: No, its removed now

**Pull request merged**